azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/invoker/storage/EventHubProcesser.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.function.invoker.storage;
+
+import com.google.gson.Gson;
+import com.microsoft.azure.eventhubs.ConnectionStringBuilder;
+import com.microsoft.azure.eventhubs.EventData;
+import com.microsoft.azure.eventhubs.EventHubClient;
+import com.microsoft.azure.eventhubs.EventPosition;
+import com.microsoft.azure.eventhubs.PartitionReceiver;
+import com.microsoft.azure.management.Azure;
+import com.microsoft.azure.management.eventhub.EventHub;
+import com.microsoft.azure.management.eventhub.EventHubNamespace;
+import com.microsoft.azure.management.resources.ResourceGroup;
+import com.microsoft.azure.management.resources.fluentcore.arm.Region;
+import com.microsoft.azure.management.storage.StorageAccount;
+import com.microsoft.azure.management.storage.StorageAccountSkuType;
+import com.microsoft.azure.maven.function.invoker.CommonUtils;
+
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+public class EventHubProcesser {
+
+    private static final String SAS_KAY_NAME = "RootManageSharedAccessKey";
+
+    private EventHubNamespace eventHubNamespace;
+    private StorageAccount storageAccount;
+    private ResourceGroup resourceGroup;
+
+    private ExecutorService executorService = Executors.newCachedThreadPool();
+    private Map<String, EventHubClient> eventHubClientMap = new HashMap<>();
+
+    public EventHubProcesser(String resourceGroupName, String namespaceName, String storageAccountName)
+            throws Exception {
+        final Azure azureClint = CommonUtils.getAzureClient();
+
+        if (azureClint.resourceGroups().contain(resourceGroupName)) {
+            resourceGroup = azureClint.resourceGroups().getByName(resourceGroupName);
+        } else {
+            resourceGroup = azureClint.resourceGroups().define(resourceGroupName).withRegion(Region.US_EAST).create();
+        }
+
+        final boolean isEventHubNamespaceExist = azureClint.eventHubNamespaces().list().stream()
```

Comment:
Why not use the same way as getting the `storageAccount`? Try to get the value first, if the return value is null, then to create.